src/xdocs/writingchecks.xml

```diff
@@ -656,20 +656,6 @@ public class MethodLimitCheck extends AbstractCheck
         There are basically only a few limits for Checkstyle:
       </p>
       <ul>
-        <li>Java
-          <a href="https://docs.oracle.com/javase/specs/jls/se8/html/jls-3.html#jls-3.5">
-            tokens (identifiers, keywords)</a>
-          should be written with
-          <a href="https://en.wikipedia.org/wiki/ASCII">ASCII</a>
-          characters ONLY,
-          no
-          <a href="https://docs.oracle.com/javase/specs/jls/se8/html/jls-3.html#jls-3.3">
-            Unicode escape</a>
-          support in keywords and no
```

Comment:
Can you show an example of keyword support?

Reply:
Are there any existing keywords that contain unicode chars?

Reply:
When I read this I thought it meant it would recognize unicode equivalent of ASCII characters. If this is not the case, then I guess this just points back to my other comment where we don't support the escapes.

Reply:
See above, thanks a lot for pointing this out.